Codecademy/gamut
# docs: fixes LinkTo and adds new docs to README

### Overview


Fixes LinkTo and adds new docs to README. Also adds inline examples to ToolTips after realizing they weren't rendering


### PR Checklist

- [x] Related to JIRA ticket: [GM-917]
- [n/a] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions


* Check out the new README
* Check some places with LinkTo
* Check out the examples in ToolTips

[GM-917]: https://skillsoftdev.atlassian.net/browse/GM-917?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
